android/src/main/java/com/theweflex/react/WeChatModule.java

```diff
@@ -66,6 +66,15 @@ public String getName() {
         return "RCTWeChat";
     }
 
+    /**
+     * fix Native module WeChatModule tried to override WeChatModule for module name RCTWeChat.
+     * If this was your intention, return true from WeChatModule#canOverrideExistingModule() bug
+     * @return
+     */
+    public boolean canOverrideExistingModule(){
```

Comment:
Do we need to add `@Override` attribute? I guess this is a method inherits from `ReactContextBaseJavaModule`